src/dbt_osmosis/core/osmosis.py

```diff
@@ -1029,6 +1030,31 @@ def remove_columns_not_in_database(
             )
         return changes_committed
 
+    @staticmethod
+    def get_prior_knowledge(
+        knowledge: Dict[str, Dict[str, Any]],
+        column: str,
+    ) -> Dict[str, Any]:
+        camel_column = re.sub("_(.)", lambda m: m.group(1).upper(), column)
+        prior_knowledge_candidates = [
```

Comment:
Lets try like this:

```python
prior_knowledge_candidates = filter(lambda k: k, [
    knowledge.get(column),
    knowledge.get(column.lower()),
    knowledge.get(camel_column),
]
```

then we can drop the `if knowledge is not None` from both of the list comprehensions

Reply:
fixed with [05c4b0f](https://github.com/z3z1ma/dbt-osmosis/pull/89/commits/05c4b0f797f720891711c60919e2b1d8fba543b0)